src/st/no0/e_stage_decoration.c

```diff
@@ -0,0 +1,119 @@
+    ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_OVL(1),
+    ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_OVL(1),
+    ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_OVL(1),
+    ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_OVL(1), ANIMSET_DRA(0)};
+
+static u8 angel_statue_left[] = {0x40, 0x01, 0xFF, 0x00};
+static u8 anim_2[] = {0x40, 0x06, 0xFF, 0x00};
+static u8 angel_statue_left_2[] = {0x40, 0x01, 0xFF, 0x00};
+static u8 angel_statue_right[] = {0x40, 0x02, 0xFF, 0x00};
+static u8 lamp_brasier[] = {0x40, 0x2F, 0xFF, 0x00};
+static u8 clock_face_1[] = {0x04, 0x22, 0x00, 0x00};
+static u8 clock_face_2[] = {0x04, 0x23, 0x00, 0x00};
+static u8 clock_face_3[] = {0x04, 0x24, 0x00, 0x00};
+static u8 window_highlight_1[] = {0x04, 0x25, 0x00, 0x00};
+static u8 window_highlight_2[] = {0x04, 0x26, 0x00, 0x00};
+static u8 window_highlight_3[] = {0x04, 0x27, 0x00, 0x00};
+static u8 window_highlight_4[] = {0x04, 0x28, 0x00, 0x00};
+static u8 red_moveable_floor[] = {0x04, 0x2A, 0x00, 0x00};
+
+static u16 zPriority[] = {
+    0x01FA, 0x0040, 0x006A, 0x006A, 0x003F, 0x0080, 0x0080,
+    0x0080, 0x0080, 0x0080, 0x0080, 0x0080, 0x0080, 0x0080,
+    0x0080, 0x0080, 0x009E, 0x009E, 0x0080, 0x0000};
+
+static u16 unk5A[] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+                      0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+static u16 palette[] = {
+    0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+static u16 drawFlags[] = {
+    FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT,
+    FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT,
+    FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT,
+    FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT,
+    FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT, FLAG_DRAW_DEFAULT};
+static u16 drawMode[] = {
+    DRAW_TPAGE,
+    DRAW_DEFAULT,
+    DRAW_DEFAULT,
+    DRAW_DEFAULT,
+    DRAW_TPAGE2 | DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_TPAGE,
+    DRAW_DEFAULT,
+    DRAW_DEFAULT,
+    DRAW_TPAGE,
+    DRAW_DEFAULT};
+static s32 flags[] = {0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0};
+static u32 facingLeft[] = {
+    false, false, false, false, false, false, false, false, false, false,
+    false, false, true,  true,  true,  true,  false, true,  true};
+static u8* animations[] = {
+    angel_statue_left,   anim_2,
+    angel_statue_left_2, angel_statue_right,
+    lamp_brasier,        clock_face_1,
+    clock_face_2,        clock_face_3,
+    window_highlight_1,  window_highlight_2,
+    window_highlight_3,  window_highlight_4,
+    clock_face_2,        clock_face_3,
+    window_highlight_2,  window_highlight_4,
+    red_moveable_floor,  red_moveable_floor,
+    window_highlight_3};
+
+void EntityStageDecoration(Entity* self) {
```

Comment:
This is quite similar to some other functions like `EntityRoomForeground` and `func_801B0958` in `nz0`

With how the data is packed and since this has a `facingLeft` I don't think this can be worked into the existing functions, or use `ObjInit` / `ObjInit2' unfortunately, so this will have to be an almost duplicate